File: SPOJ/370.py
a=['2','3','4','5','6','7','8','9']
def check(n):
	global a
	for i in a:
		if i in str(n):
			return 0
	return 1
x=input()
while x:
	x=x-1
	y=input()
	z=y
	while 1:
		if check(y)==1:
			break
		y+=z
	print y

File: SPOJ/370.c
#include<stdio.h>
int check(int );
int main()
{
	int test_cases;
	scanf("%d",&test_cases);
	while(test_cases--)
	{
		int n,x;
		scanf("%d",&n);
		x=n;
		while(1)
		{
			if(check(n)==1)
				break;
			n+=x;
		}	
		printf("%d\n",n);
	}
	return 0;
}
int check(int n)
{
	while(n!=0)
	{
		if(n%10==1 || n%10==0)
		{
			n/=10;
			continue;
		}
		return 0;
	}
	return 1;
}
